internal/commands/fetch.go

```diff
@@ -24,7 +24,7 @@ type FetchName struct {
 
 var ErrInvalidFetchName = errors.New("invalid fetch name")
 
-var fetchNameRegex = regexp.MustCompile(`^([\w\-0-9]+(/[\w\-0-9]+)+)(:(.+))?$`)
+var fetchNameRegex = regexp.MustCompile(`^([\w\-0-9]+(/[\w\-0-9]+){2,})(:(.+))?$`)
```

Comment:
Related to the last discussions and changes of a sanitized name, and as \w means lower/uppercase letters and "_",  
only "[a-z]", "-", "[0-9]" should be allowed in name parts to be a correct fetch name.

Reply:
I know. I'm working on that. It'll come in the next PR.